test/test_ordered_dict.jl

```diff
@@ -406,4 +406,9 @@ using OrderedCollections, Test
         @test collect(values(sdv)) == collect('q':'z')
     end
 
+    @testset "Pull Request #11" begin
```

Comment:
Not to be too pedantic, but could you briefly summarize the functionality being tested in the test set as its name?